edg_core/Blocks.py

```diff
@@ -290,6 +292,31 @@ def _populate_def_proto_block_contents(self, pb: BaseBlockEdgirType) -> BaseBloc
 
     return pb
 
+  def _populate_def_proto_description(self, pb: BaseBlockEdgirType) -> BaseBlockEdgirType:
+
+    description = self.description
+    size = len(description)
+
+    stringStart = 0
+
+    for varIndex, char in enumerate(description):
```

Comment:
I don't think you use `size`? And in which case you only reference `self.description` once and can move it directly into the enumerate?

Also, an alternative architecture would be to use string find, instead of iterating over each character.